python/eos/figure/data.py

```diff
@@ -95,7 +95,7 @@ def labels(self, variables):
             return [p[dist].latex() for dist in variables]
         elif self._type == 'prediction':
             o = eos.Observables()
-            return ["$" + o[dist].latex() + "$" for dist in variables]
+            return ["$" + o[dist.split(";")[0]].latex() + "$" for dist in variables]
```

Comment:
Hm. Is this a qualified name? If so, we should probably use
```py
dist = eos.QualifiedName(dist)
...
return ["$" + o[str(dist)].latex() + "$" for dist in variables]
```

Reply:
I see. That's a bug in whatever produces that particular file. It should read:
```
CKM::Vus[...];form-factors=KSvD2025,model=CKM,n-resonances-0p=2,n-resonances-1m=2
```
i.e., the kinematics should be added to the name part.

Reply:
Yes let's do this for now, since even with the square bracket in the right place `str(eos.QualifiedName(dist))` contains the square brackets and so it still doesn't work